### PR TITLE
feat: add mainnet channel, remove betas

### PR DESCRIPTION
**Pull request by sdankel**
- adds mainnet channel
- removes betas 1-5, and devnet

### CODE REVIEW

**Comment by fuel-service-user** [comment]
### [LCOV](https://github.com/marketplace/actions/report-lcov) of commit [<code>4a57b2b</code>](677/commits/4a57b2b60a695146123e81f95f8560ddf559985e) during [CI #2059](../actions/runs/11352465126)
<pre>Summary coverage rate:
  lines......: 81.8% (2294 of 2805 lines)
  functions..: 44.4% (363 of 818 functions)
  branches...: 58.1% (264 of 454 branches)

Files changed coverage rate: n/a</pre>

File: src/channel.rs [hunk 1]
```diff
@@ -51,13 +43,7 @@ pub struct Package {
 }
 
 pub fn is_beta_toolchain(name: &str) -> bool {
```
Comment (kayagokalp) [inline src/channel.rs:45]: Should we rename this into something like `is_reserved_channel`? Given that we do not have any beta networks right now.

```suggestion
pub fn is_reserved_toolchain(name: &str) -> bool {
```

**Review by kayagokalp — COMMENTED** [review]
Should we first merge https://github.com/FuelLabs/fuelup/pull/676 and see if this actually works? Right now building this branch and executing `fuelup default mainnet` is not working for me. I think reserved channel logic is working but since we lack the channel toml it is not installing anything?

File: src/channel.rs (hunk 1 above)
Comment (sdankel) [reply src/channel.rs]: I changed it to `is_dateless_distributed_toolchain` since "reserved" doesn't fully describe it

**Review by kayagokalp — DISMISSED** [review]
🚀

**Review by kayagokalp — DISMISSED** [review]

**Review by kayagokalp — APPROVED** [review]

**Pull request merged**